.github/workflows/create-release-on-branch-changes.yml

```diff
@@ -85,6 +85,21 @@ jobs:
           body: ${{ env.RELEASE_DESCRIPTION }}
           files: ${{ env.DIST_FILENAME }}
 
+      - name: Write release information in artifact file
+        run: |
+            printf '{
+              "release_branch": "${{ env.RELEASE_BRANCH }}",
+              "dependency_package": "danskernesdigitalebibliotek/dpl-design-system",
+              "release_download_url": "${{ env.DOWNLOAD_URL }}"
+            }' >> release-info.json
+        env:
+          DOWNLOAD_URL: "${{ github.event.repository.html_url }}/releases/download/${{ env.TAG_NAME }}/${{ env.DIST_FILENAME }}"
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: release-info.json
+          path: ./release-info.json
+
```

Comment:
My idea of creating a separate workflow () that triggers a remote pr in dpl-cms was to have a separation of concerns. Instead if letting ..... grow I thought it would be a cleaner to have the functionality that requests a pr in a separate workflow. In that way it is more clear what the responsibility is for the two workflows and we do not have one workflow growing unresonable big.
In theory we could copy [this block](https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/b34c41e26de0e534e4bc54925635b58cb16f8248/.github/workflows/create-release-on-branch-changes.yml#L22-L30) in the other workflow and then we would not need to pass data.
I'll try to implement dispatching of the other workflow instead of passing context in an asset, but honestly I think this method also works and I do not see the greater need for doing so,.

Reply:
Workflow dispatch implemented instead of json asset upload